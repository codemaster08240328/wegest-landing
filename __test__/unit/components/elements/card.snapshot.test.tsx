import renderer from 'react-test-renderer'
import Card from '@/components/elements/card'

describe('<Card> snapshot', () => {
  it('renders with props', () => {
    expect(
      renderer.create(
        <Card
          raised
          icon='email'
          className='class-name'
          title='card test'
          date='2021-04-09'
        >
          Hello World!
        </Card>,
      ),
    ).toMatchSnapshot()
  })
})
