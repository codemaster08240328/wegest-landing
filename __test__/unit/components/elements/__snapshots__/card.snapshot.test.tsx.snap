// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card> snapshot renders with props 1`] = `
<div
  className="MuiPaper-root MuiCard-root makeStyles-root-1 class-name MuiPaper-elevation8 MuiPaper-rounded"
  data-testid="el-card"
>
  <div
    className="MuiCardContent-root WithStyles(ForwardRef(CardContent))-root-3"
  >
    <div
      className="MuiBox-root MuiBox-root-4 makeStyles-title-2"
    >
      <span
        className="MuiBox-root MuiBox-root-5"
      >
        <span
          aria-hidden={true}
          className="material-icons MuiIcon-root"
          data-testid="el-icon"
        >
          email
        </span>
        <p
          className="MuiTypography-root MuiTypography-body1 MuiTypography-colorSecondary"
        >
          card test
        </p>
      </span>
      <p
        className="MuiTypography-root MuiTypography-body1 MuiTypography-colorSecondary"
      >
        2021-04-09
      </p>
    </div>
    <div
      className="MuiBox-root MuiBox-root-6"
    >
      Hello World!
    </div>
  </div>
</div>
`;
